core/css/decanter-grid.css

```diff
@@ -12,100 +12,186 @@
       padding-right: 3rem;
       padding-left: 3rem; } }
 
-@media only screen and (min-width: 768px) and (max-width: 1023px) {
-  .grid--item:last-child:nth-child(odd) {
-    width: calc(1 / 1 * 100% - (0px)); } }
-
 .grid--3-of-12 {
   width: 100%; }
+  @media only screen and (min-width: 0) {
+    .grid--3-of-12 {
+      width: calc(25% - 15px); } }
+  @media only screen and (min-width: 576px) {
+    .grid--3-of-12 {
+      width: calc(25% - 15px); } }
   @media only screen and (min-width: 768px) {
     .grid--3-of-12 {
-      width: calc(1 / 2 * 100% - (10px)); } }
+      width: calc(25% - 15px); } }
```

Comment:
Did this 3 of 12 size change to 6 of 12 at the 768 screen size in the previous version? Looks like maybe. I will need to check and revise.